force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -12082,7 +12082,7 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <categories>Payment, Error</categories>
         <language>en_US</language>
         <protected>true</protected>
-        <shortDescription>Error when user does not have a permission to change an RD originated in Elevate</shortDescription>
+        <shortDescription>Error when user does not have a permission to change a Payment originated in Elevate</shortDescription>
```

Comment:
This Short Description is too long, suggested change: "Error when user does not have permission to change a Payment from Elevate"